src/DIPS.Xamarin.UI/Controls/Sheet/SheetBehavior.cs

```diff
@@ -482,6 +497,8 @@ public Task AfterRemoval()
         /// </summary>
         public static readonly BindableProperty CloseOnOverlayTappedProperty = BindableProperty.Create(nameof(CloseOnOverlayTapped), typeof(bool), typeof(SheetBehavior), true);
 
+        private double m_originalPosition = -1;

```

Comment:
I guess this will work with multiple sheets on the same page too? As you store it in the Behavior?

Reply:
Should work, I can test it